includes/reports/overview/list-table/class-sensei-reports-overview-list-table-courses.php

```diff
@@ -85,10 +86,21 @@ public function get_columns() {
 				esc_html( count( $all_course_ids ) )
 			),
 			'last_activity'      => __( 'Last Activity', 'sensei-lms' ),
+			'enrolled'           => sprintf(
+				// translators: Placeholder value is the total average progress for all courses.
```

Comment:
As I can understand, it is not the total average progress, but the total number of enrollments across all courses.

Reply:
Fixed 19ba9b92150eccde57b1fc65ae2575854852764e